### PR TITLE
Add libfuse util strtol wrapper

### PR DESCRIPTION
**Pull request by joannekoong**
Add a wrapper around strtol for more rigorous error checking and convert uses of atoi and strtol to use this instead.

### CODE REVIEW

File: lib/util.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include <stdlib.h>
+#include <errno.h>
+
+#include "util.h"
+
+long libfuse_strtol(const char *str)
```
Comment (trapexit) [inline lib/util.c:6]: Is this not confusing? Forcing a signed type to be effectively unsigned?

Comment (bsbernd) [reply lib/util.c:6]: For me as well, the wrapper I had in my mind is more like

int libfuse_strtol(const char *str, long *res);

Return is then 0/error-code and res provides the result.

@joannekoong Sorry for my late reply, was concentrating on fuse-uring. Do want to make the update above, or shall I do that for you?

Comment (joannekoong) [reply lib/util.c:6]: Ah I didn't realize "strtol" also takes in negative numbered strings. I'll fix this up and resubmit

**Comment by bsbernd** [comment]
Thank you very much @joannekoong !

**Pull request merged**